app/(chat)/api/chat/route.ts

```
@@ -1,167 +1,108 @@
-import { requestSuggestions } from '@/lib/ai/tools/request-suggestions';
-import { getWeather } from '@/lib/ai/tools/get-weather';
-import { isProductionEnvironment } from '@/lib/constants';
-import { myProvider } from '@/lib/ai/providers';
+  experimental_createMCPClient as createMCPClient,
+  InvalidToolArgumentsError,
+  NoSuchToolError,
+  smoothStream,
+  streamText,
+  ToolExecutionError
+} from 'ai';
+import { NextRequest } from 'next/server';
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)